vscode-lean4/package-lock.json

```diff
@@ -15,7 +15,7 @@
 				"cheerio": "^1.0.0-rc.10",
 				"mobx": "5.15.7",
 				"semver": "=7.3.5",
-				"vscode-languageclient": "=7.0.0"
+				"vscode-languageclient": "^8.0.0-next.14"
```

Comment:
Please revert this language client version change

Reply:
I would also be happy to merge that one first and then review this one once it is rebased on the new version, which would save the reverting.

Reply:
You can find this on: #162